src/hp/magicsock/conn.rs

```diff
@@ -140,6 +149,7 @@ impl Deref for Conn {
 pub struct Inner {
     actor_sender: flume::Sender<ActorMessage>,
     /// Sends network messages.
+    // TODO: send transmits *together* with their destination
```

Comment:
the transmit includes the destination, what do you mean with this comment?

Reply:
I meant that I'd like the lookup of the real destination to be at the time of `poll_send` and include the destination along.  Because that's where the error should be propagated.  To be fair I'm not sure if that makes sense anymore so i'll delete the comment.